mender/install_mender.sh

```diff
@@ -0,0 +1,75 @@
+#!/bin/bash
+set -e
+
+if [ "$EUID" -ne 0 ]
+  then echo "Please run with sudo"
+  exit
+fi
+
+
+
+function get_mender()
+{
+   [ ! -d ${HOME}/mender-connect ] && git clone -b 2.0.1 https://github.com/mendersoftware/mender-connect.git ./mender-connect
```

Comment:
Here you are verifying the presence in HOME and cloning the repo in `${HOME}/system-dependencies/mender`.
```suggestion
   [ ! -d ${HOME}/mender-connect ] && git clone -b 2.0.1 https://github.com/mendersoftware/mender-connect.git ${HOME}/mender-connect
```

Reply:
Or
```suggestion
   [ ! -d ./mender-connect ] && git clone -b 2.0.1 https://github.com/mendersoftware/mender-connect.git ./mender-connect
```